doc/source/tutorial/tutorial_presenting_larray_objects.ipyml

```diff
@@ -1,492 +0,0 @@
-
-
-- markdown: |
-    ## Session
-    
-    A ``Session`` object is a dictionary-like object used to gather several arrays, axes and groups. 
-    A session is particularly adapted to gather all input objects of a model or to gather the output arrays from different scenarios. Like with arrays, it is possible to associate metadata to sessions.
-
-
-- markdown: |
-    ### Creating Sessions
-    
-    To create a session, you can first create an empty session and then populate it with arrays, axes and groups:
-
-
-- code: |
-    gender = Axis("gender=Male,Female")
-    time = Axis("time=2013..2017")
-    
-    # create an empty session
-    demography_session = Session()
-    
-    # add axes to the session
-    demography_session.gender = gender
-    demography_session.time = time
-    
-    # add arrays to the session
-    demography_session.population = zeros((gender, time))
-    demography_session.births = zeros((gender, time))
-    demography_session.deaths = zeros((gender, time))
-    
-    # add metadata after creation
-    demography_session.meta.title = 'Demographic Model of Belgium'
-    demography_session.meta.description = 'Models the demography of Belgium'
-    
-    # print content of the session
-    print(demography_session.summary())
-
-
-- markdown: |
-    or you can create and populate a session in one step:
-
-
-- code: |
-    gender = Axis("gender=Male,Female")
-    time = Axis("time=2013..2017")
-    
-    demography_session = Session(gender=gender, time=time, population=zeros((gender, time)), 
-                         births=zeros((gender, time)), deaths=zeros((gender, time)), 
-                         meta=Metadata(title='Demographic Model of Belgium', description='Modelize the demography of Belgium'))
-    
-    # print content of the session
-    print(demography_session.summary())
-
-
-- markdown: |
-    <div class="alert alert-warning">
-    
-    **Warning:**
-      <ul>
-        <li>Contrary to array metadata, saving and loading session metadata is supported for
-            all current session file formats: Excel, CSV and HDF (.h5).</li>
-        <li>Metadata is not kept when actions or methods are applied on a session
-            except for operations modifying a session in-place, such as: `s.arr1 = 0`.
-            Do not add metadata to a session if you know you will apply actions or methods
-            on it before dumping it.</li>
-      </ul>
-    
-    </div>
-
-
-- markdown: |
-    ### More on Session objects
-    
-    To know how to save and load sessions in CSV, Excel or HDF format, please refer to the [Loading and Dumping Sessions](tutorial_sessions.ipynb#Loading-and-Dumping-Sessions) section of the tutorial.
-    
-    To see how to work with sessions, please read the [Working With Sessions](tutorial_sessions.ipynb#Working-With-Sessions) section of the tutorial.
-    
-    Finally, see the [Session](../api.rst#session) section of the API Reference to explore all methods of Session objects.
```

Comment:
There is no link to the API section in Getting started. I would add one.

Reply:
OK